# 初始位姿
InitialPose:
  position_x: 0
  position_y: 0
  position_z: 0
  orientation_x: 0
  orientation_y: 0
  orientation_z: 0
  orientation_w: 1

# 零食台位姿
SnackDesk:
  position_x: 0.40
  position_y: -2.33
  position_z: 0
  orientation_x: 0
  orientation_y: 0
  orientation_z: 1
  orientation_w: 0
  
# 饮料台位姿
DrinkDesk:
  position_x: 0.60
  position_y: -2.30
  position_z: -0
  orientation_x: 0.00152
  orientation_y: 0.00167
  orientation_z: -0.7173561
  orientation_w: 0.6967067


# 右服务台位姿
RightServiceDesk:
  position_x: 1.92
  position_y: -0.55
  position_z: 0
  orientation_x: 0
  orientation_y: 0
  orientation_z: 0
  orientation_w: 1


# 左服务台位姿
LeftServiceDesk:
  position_x: 1.97
  position_y: -2.00
  position_z: 0
  orientation_x: 0
  orientation_y: 0
  orientation_z: 0
  orientation_w: 1
